### ISSUE TITLE
Incorrect dependancy range for @dessert-box/core

### ISSUE THREAD

**Issue by ehyland** [TheMightyPenguin/dessert-box#17]
Hit this error after updating `@desert-box/react` 

```
WARNING in ./node_modules/.pnpm/@dessert-box+react@0.3.0_react@18.1.0/node_modules/@dessert-box/react/dist/dessert-box-react.esm.js 25:17-34
export 'composeClassNames' (imported as 'composeClassNames') was not found in '@dessert-box/core' (possible exports: extractAtomsFromProps, getClassName)
```

I think you just need to bump the despondency on core to `^0.2.0` [here](https://github.com/TheMightyPenguin/dessert-box/blob/main/packages/react/package.json#L26)

**Comment by jgrancher** [comment]
For reference, I also had a similar error with version `0.3.0`:

```
TypeError: core.composeClassNames is not a function
```

Installing `@desert-box/react@0.2.0` instead made it go away 👍

**Comment by TheMightyPenguin** [comment]
@ehyland @jgrancher Just published version `0.4.0` with a fix for this, sorry about it! Will add some extra tests to make sure issue like this don't go trough next time!

**Comment by TheMightyPenguin** [comment]
Closing as this was fixed.
